Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx

```diff
@@ -185,7 +185,7 @@ Tags.propTypes = {
 };
 
 Tags.defaultProps = {
-    addTagsPlaceholder: "Add Tags",
+    addTagsPlaceholder: "{Localization.get("addTagsPlaceholder")}",
     searchTagsPlaceholder: "Begin typing to search tags",
```

Comment:
Also, could be nice to do this other one in the same go...

Reply:
you mean this element on line 188? **searchTagsPlaceholder**

Reply:
Yeah, also, should'nt the double-quotes be removed in this context ?

Reply:
Yes, I also agree

Reply:
@valadas but i need to edit resources now and i cant find the right file address in the source and i don't know how to edit more files for 1 pull request!